NEWS.md

```diff
@@ -1,6 +1,45 @@
 
 # rlang (development version)
 
+* You can now use glue syntax to unquote on the LHS of `:=`. This
```

Comment:
Maybe mention that if you use this in a package you'll also need to import glue?